packages/dev/src/prompts.ts

```diff
@@ -0,0 +1,10 @@
+import { AgentFunctionDefinition } from "@evo-ninja/agent-utils";
+
+export const INITIAL_PROMP = (functions: AgentFunctionDefinition[]) =>
```

Comment:
Are the functions here necessary?

Reply:
Whoops, no that was an artifact of when I had them in the initial prompt. It actually performed better when i left them out of the agent message, and only had them in the `functions` array.